File: pertemuan1/captcha01.js
let captcha = Math.floor(1000 + Math.random() * 9000);
console.log(`Kode CAPTCHA: ${captcha}`);

let inputUser = parseInt(prompt("Masukkan kode CAPTCHA: "));

if (inputUser === captcha) {
  console.log("Verifikasi berhasil.");
} else {
  console.log("Verifikasi gagal.");
}

File: pertemuan1/captcha01.py
import random
   
captcha = random.randint(1000, 9999)
print(f"Kode CAPTCHA: {captcha}")
   
input_user = int(input("Masukkan kode CAPTCHA: "))
   
if input_user == captcha:
    print("Verifikasi berhasil.")
else:
    print("Verifikasi gagal.")